embrace-android-sdk/api/embrace-android-sdk.api

```diff
@@ -42,9 +42,6 @@ public final class io/embrace/android/embracesdk/Embrace : io/embrace/android/em
 	public fun getSessionProperties ()Ljava/util/Map;
 	public fun getSpan (Ljava/lang/String;)Lio/embrace/android/embracesdk/spans/EmbraceSpan;
 	public fun getTraceIdHeader ()Ljava/lang/String;
-	public fun getTracer ()Lio/opentelemetry/api/trace/Tracer;
-	public fun getTracer (Ljava/lang/String;)Lio/opentelemetry/api/trace/Tracer;
-	public fun getTracer (Ljava/lang/String;Ljava/lang/String;)Lio/opentelemetry/api/trace/Tracer;
```

Comment:
I'm assuming that these haven't been released as part of our public API yet

Reply:
Nope. Just added them last week 😅